HACKING.md

```diff
@@ -75,30 +75,15 @@ SENTRY_DSN=<DSN_FROM_ABOVE>
 
 The application will be reporting errors to your `sentry.io` project from now on.
 
-
-## Building CSS
-
-For working on [Sass files](static/css), you may want to dynamically watch for changes to rebuild the CSS whenever something changes.
-
-To setup the watcher, open a new terminal window and run:
-
-``` bash
-./run watch
-```
-
 ## Testing
 
 ``` bash
-./run test
+dotrun test
```

Comment:
This is the first mention of `dotrun` should we not have a section about installing and what it is?